syntax = "proto3";
import "PlayerInfo.proto";

message ChangeRoleC2S {
    Role role = 1;
}